packages/devtools/src/common.css

```diff
@@ -0,0 +1,218 @@
+@charset "UTF-8";
```

Comment:
Please remove the compiled css from source code.

Reply:
It seems that there is no declaration for *.css files in .gitignore, should we add one here? Please provide a detailed answer.